tests/pkg_semtech-loramac/README.md

```diff
@@ -147,4 +147,14 @@ The node will also print the data received:
 ```
     > loramac tx test
     Data received: This is RIOT!
-```
\ No newline at end of file
+```
+
+### The Things Network
```

Comment:
In fact, I would move this before the MQTT related stuff. Maybe the README should be reorganized as follows:

Keep section "Using the shell" but refactor it slightly with:
- a subsection "OTAA activation"
- a subsection "ABP activation", where information is given with required keys/EUI and the trick with the RX2 setting
- a subsection with sending receiving messages
- a subsection "Other shell commands" with other setting commands (DR, ADR, etc) plus the shell help

What do you think @jia200x ?

Reply:
yes, I agree. I can use this same PR